wire-ios-sync-engine/Source/SessionManager/SessionFactories.swift

```diff
@@ -145,6 +145,7 @@ open class AuthenticatedSessionFactory {
 
         let userSession = userSessionBuilder.build()
         userSession.setup(
+            apiVersion: nil,
```

Comment:
This PR also fixes a little bug where the old network stack (operation loop + request strategies) didn't know which api version to use. This is because when multibackend support is enabled, we are no longer storing the api version in `BackendInfo`. You'll see later that code has changed. Here we pass `nil` because this is the legacy code path and it's not needed here.